packages/screens/AdministrationDashboard/component/GuardiansBox.tsx

```diff
@@ -0,0 +1,241 @@
+                  <SVG source={dotSVG} />
+                </View>
+              </View>
+              <Box notched style={herosLisBox}>
+                <TouchableOpacity
+                  onPress={() =>
+                    setIsListOpen((old: any) => (old === index ? null : index))
+                  }
+                  style={toggleBox}
+                >
+                  <BrandText style={[fontSemibold14]}>
+                    Genesis Guardians
+                  </BrandText>
+                  <View>
+                    {isListOpen === index ? (
+                      <SVG source={downSVG} />
+                    ) : (
+                      <SVG source={upSVG} />
+                    )}
+                  </View>
+                </TouchableOpacity>
+                {isListOpen === index ? (
+                  <Box
+                    notched
+                    style={{
+                      height: 250,
+                      width: "100%",
+                      borderWidth: 1,
+                      borderColor: gradientColorLightBlue,
+                    }}
+                  >
+                    <View style={{ margin: 10 }}>
+                      <SearchInput handleChangeText={() => {}} />
+                      <FlatList
+                        showsVerticalScrollIndicator={false}
+                        data={Array(10).fill(0)}
+                        renderItem={({ item }) => (
+                          <View style={listToggle}>
+                            <Image
+                              style={{
+                                width: 28,
+                                height: 28,
+                              }}
+                              source={avaPNG}
+                            />
+                            <BrandText
+                              style={[
+                                fontSemibold16,
+                                { marginLeft: 15, marginRight: 10 },
+                              ]}
+                            >
+                              Meebits
+                            </BrandText>
+                            <SVG source={checkBadgeSVG} />
+                          </View>
+                        )}
+                        keyExtractor={(item: any) => item.id}
+                        style={{
+                          marginTop: 12,
+                          marginLeft: 5,
+                        }}
+                        contentContainerStyle={{
+                          height: 180,
+                        }}
+                      />
+                    </View>
+                  </Box>
+                ) : (
+                  <ImageBackground
+                    source={guardianPng}
+                    imageStyle={{ borderRadius: 16 }}
+                    style={imageBgStyle}
+                  >
+                    <TouchableOpacity
+                      onPress={() => {
+                        const old = [...guardiansHandler];
```

Comment:
Except we need to keep original value of `guardiansHandler`, if not it's safe to modify directly `guardiansHandler` here, dont need to use var `old`